files/renew_token.sh

```diff
@@ -6,17 +6,16 @@ set -xe
 CONFIG=/usr/local/openresty/nginx/conf/nginx.conf
 AUTH=$(grep  X-Forwarded-User $CONFIG | awk '{print $4}'| uniq|tr -d "\n\r")
 
+set +x
```

Comment:
Is this required?

Reply:
Yes to hide the echo command, otherwise it will be displayed in the logs.

Reply:
You mean the
`echo $TOKEN > /usr/local/openresty/nginx/token.txt` ?
Then whey not do set +x directly before that?

Reply:
The is a https://github.com/Lotto24/aws-ecr-http-proxy/blob/4afdffc4e7878a7ca2e9775e42ffd52b83423b0b/files/renew_token.sh#L3 in the main branch.

Reply:
The set -x is okay in general IMO.
My point is that you're hiding more of the output than necessary

Reply:
Clear, move the set +x to only HIDE a specific part.